csrank/choicefunction/feta_choice.py

```diff
@@ -0,0 +1,80 @@
+import functools
+
+import torch.nn as nn
+
+from csrank.choicefunction.choice_functions import SkorchChoiceFunction
+from csrank.modules.object_mapping import DenseNeuralNetwork
+from csrank.modules.scoring.feta import FETAScoring
+
+
+class FETAChoiceFunction(SkorchChoiceFunction):
+    """A variable choice estimator based on the FETA-Approach.
+
+    Parameters
+    ----------
+    n_hidden_set_layers : int
+        The number of hidden layers that should be used for the ``DeepSet``
+        context embedding.
+
+    n_hidden_set_untis : int
+        The number of units per hidden layer that should be used for the
+        ``DeepSet`` context embedding.
+
+    n_hidden_joint_layers : int
+        The number of hidden layers that should be used for the utility
+        function that evaluates each object in the aggregated context.
+
+    n_hidden_joint_units : int
+        The number of units per hidden layer that should used for the utility
+        function that evaluates each object in the aggregated context.
```

Comment:
The docstring is copied from FATE apparently.

Reply:
I meant to revisit the docstrings but then forgot to actually do it. I should have caught that in self-review. Thanks for noticing.

Reply:
Changed.